.github/workflows/pr-review.yml

```diff
@@ -0,0 +1,22 @@
+name: PR review
+# This workflow is triggered when a PR review is submitted. It checks if the review is approved and runs a script to add labels to the PR based on the review.
+on:
+  pull_request_review:
+    types:
+      - submitted
```

Comment:
Hmm, I think that the `pull_request_review` trigger only provides a read-only `GITHUB_TOKEN` - I guess it would not be able to re-label the PR in that case, right? 🤔 

Thinking more about it, I'm also wondering if there are some security concerns if we were to give the workflow additional permissions. 🤔

Reply:
We can add permissions by workflow right? that would prevent us to change the default permissions and keep them read only for other workflows

Reply:
yes, I think you're right  - let's use 
```     
      contents: read
      pull-requests: write
```

on the job as that's what another workflow is using already. Then I think this PR is ready to go. 

The overall workflow permission can be `contents: read`.